scripts/ram_percentage.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+source "$CURRENT_DIR/helpers.sh"
+
+ram_percentage_format="%3.1f%%"
+
+print_ram_percentage() {
+  ram_percentage_format=$(get_tmux_option "@ram_percentage_format" "$ram_percentage_format")
+
+  if command_exists "free"; then
+    free | awk -v format="$ram_percentage_format" '$1 == "Mem:" {printf(format, 100*$3/$2)}'
+  fi
```

Comment:
@ctjhoa doyou think we should have an `else echo "free not found"` message? I think not. Would be better to have an `elif command_exists <some_other_cmd>`...

Reply:
You can also get memory information from `/proc/meminfo` as a backup, although I'm not sure whether it is cross-distribution.

Reply:
> **free**  displays the total amount of free and used physical and swap memory in the system, as well as the buffers and caches used by  the  kernel.  The  information  is  gathered by parsing `/proc/meminfo`.

So it looks like `free` should be supported anywhere `/proc/meminfo` is available.

Reply:
Another alternative is `vmstat`, but I'm pretty sure `free` should be available on vast majority Linux distros.

Reply:
I'm ok without `else` but we should precise `free` requirement in the README.

Reply:
done now